src/helix/endpoints/eventsub/get_conduit_shards.rs

```diff
@@ -0,0 +1,221 @@
+//! Gets a list of all shards for a conduit.
+//! [`get-conduit-shards`](https://dev.twitch.tv/docs/api/reference/#get-conduit-shards)
+
+use super::*;
+use crate::eventsub;
+use helix::RequestGet;
+
+/// Query Parameters for [Get Conduit Shards](super::get_conduit_shards)
+///
+/// [`get-conduit-shards`](https://dev.twitch.tv/docs/api/reference/#get-conduit-shards)
+#[derive(PartialEq, Eq, Serialize, Clone, Debug, Default)]
+#[cfg_attr(feature = "typed-builder", derive(typed_builder::TypedBuilder))]
+#[must_use]
+#[non_exhaustive]
+pub struct GetConduitShardsRequest<'a> {
+    /// Conduit ID.
+    #[cfg_attr(feature = "typed-builder", builder(default, setter(into)))]
+    #[cfg_attr(feature = "deser_borrow", serde(borrow = "'a"))]
+    pub conduit_id: Cow<'a, str>,
+
+    /// Status to filter by.
+    #[cfg_attr(feature = "typed-builder", builder(default, setter(into)))]
+    pub status: Option<eventsub::ShardStatus>,
+
+    /// The cursor used to get the next page of results. The pagination object in the response contains the cursor’s value.
+    #[cfg_attr(feature = "typed-builder", builder(default, setter(into)))]
+    #[cfg_attr(feature = "deser_borrow", serde(borrow = "'a"))]
+    pub after: Option<Cow<'a, helix::CursorRef>>,
+}
+
+/// Return Values for [Get Conduit Shards](super::get_conduit_shards)
+///
+/// [`get-conduit-shards`](https://dev.twitch.tv/docs/api/reference/#get-conduit-shards)
+#[derive(PartialEq, Eq, Deserialize, Serialize, Debug, Clone)]
+#[cfg_attr(feature = "deny_unknown_fields", serde(deny_unknown_fields))]
+#[non_exhaustive]
+pub struct ConduitShards {
+    /// List of information about a conduit's shards.
+    shards: Vec<eventsub::ShardResponse>,
+}
+
+impl Request for GetConduitShardsRequest<'_> {
+    type Response = Vec<eventsub::ShardResponse>;
+
+    const PATH: &'static str = "eventsub/conduits/shards";
+    #[cfg(feature = "twitch_oauth2")]
+    const SCOPE: twitch_oauth2::Validator = twitch_oauth2::validator![];
+}
+
+impl RequestGet for GetConduitShardsRequest<'_> {
+    fn parse_inner_response(
+        request: Option<Self>,
+        uri: &http::Uri,
+        response: &str,
+        status: http::StatusCode,
+    ) -> Result<helix::Response<Self, Self::Response>, helix::HelixRequestGetError>
+    where
+        Self: Sized,
```

Comment:
is this really needed? Looks like it doesn't do anything special that the default impl doesn't do

Reply:
You're right - I've removed it & it still works as expected & force pushed